pyvista/core/__init__.py

```diff
@@ -24,11 +24,12 @@
 from .filters import (
     CompositeFilters,
     DataSetFilters,
+    ImageDataFilters,
     PolyDataFilters,
     UniformGridFilters,
```

Comment:
Do people rely on `pv.UniformGridFilters` in the wild? Or is it just better to be safe than saying sorry?

Reply:
I really, really hope no one is using this. This is more of a better safe than sorry and putting off the namespace cleanup for a follow up PR